packages/tcore-console/src/Components/Icon/Icon.types.ts

```diff
@@ -112,108 +113,170 @@ enum EIcon {
-  "hdd": require("../../../assets/icons/hdd.svg"),
-  "home": require("../../../assets/icons/home.svg"),
-  "image": require("../../../assets/icons/image.svg"),
-  "info-circle": require("../../../assets/icons/info-circle.svg"),
-  "io": require("../../../assets/icons/io.svg"),
-  "key": require("../../../assets/icons/key.svg"),
-  "lightbulb": require("../../../assets/icons/lightbulb.svg"),
-  "link": require("../../../assets/icons/link.svg"),
-  "linux": require("../../../assets/icons/linux.svg"),
-  "list": require("../../../assets/icons/list.svg"),
-  "lock-open": require("../../../assets/icons/lock-open.svg"),
-  "lock": require("../../../assets/icons/lock.svg"),
-  "markdown": require("../../../assets/icons/markdown.svg"),
-  "memory": require("../../../assets/icons/memory.svg"),
-  "microchip": require("../../../assets/icons/microchip.svg"),
-  "minus": require("../../../assets/icons/minus.svg"),
-  "mountain": require("../../../assets/icons/mountain.svg"),
-  "network-wired": require("../../../assets/icons/network-wired.svg"),
-  "pause": require("../../../assets/icons/pause.svg"),
-  "pencil-alt": require("../../../assets/icons/pencil-alt.svg"),
-  "play": require("../../../assets/icons/play.svg"),
-  "plus-circle": require("../../../assets/icons/plus-circle.svg"),
-  "plus": require("../../../assets/icons/plus.svg"),
-  "puzzle-piece": require("../../../assets/icons/puzzle-piece.svg"),
-  "question-circle": require("../../../assets/icons/question-circle.svg"),
-  "raspberry-pi": require("../../../assets/icons/raspberry-pi.svg"),
-  "redo": require("../../../assets/icons/redo.svg"),
-  "rock-pi": require("../../../assets/icons/rock-pi.svg"),
-  "scroll": require("../../../assets/icons/scroll.svg"),
-  "search": require("../../../assets/icons/search.svg"),
-  "shapes": require("../../../assets/icons/shapes.svg"),
-  "sign-out-alt": require("../../../assets/icons/sign-out-alt.svg"),
-  "snowflake": require("../../../assets/icons/snowflake.svg"),
-  "spinner": require("../../../assets/icons/spinner.svg"),
-  "star": require("../../../assets/icons/star.svg"),
-  "stopwatch": require("../../../assets/icons/stopwatch.svg"),
-  "store": require("../../../assets/icons/store.svg"),
-  "sync-alt": require("../../../assets/icons/sync-alt.svg"),
-  "tag": require("../../../assets/icons/tag.svg"),
-  "tcore": require("../../../assets/icons/tcore.svg"),
-  "temperature-high": require("../../../assets/icons/temperature-high.svg"),
-  "th-large": require("../../../assets/icons/th-large.svg"),
-  "times": require("../../../assets/icons/times.svg"),
-  "trash-alt": require("../../../assets/icons/trash-alt.svg"),
-  "user-alt": require("../../../assets/icons/user-alt.svg"),
-  "wifi": require("../../../assets/icons/wifi.svg"),
-  "window-maximize": require("../../../assets/icons/window-maximize.svg"),
-  "windows": require("../../../assets/icons/windows.svg"),
-}
+  alpine: lazy(() => import("../../../assets/icons/alpine.svg?react")),
```

Comment:
This was edited with search/replace, be careful since the script to generate icons needs fixing/tests